CMakeLists.txt

```diff
@@ -223,7 +223,7 @@ else()
                 add_subdirectory(src/tbbmalloc_proxy)
             endif()
         endif()
-        if (APPLE)
+        if (APPLE OR NOT BUILD_SHARED_LIBS)
             message(STATUS "TBBBind build targets are disabled due to unsupported environment")
```

Comment:
There are two tests added through `tbb_add_tbbbind_test`, e.g.: https://github.com/oneapi-src/oneTBB/blob/9052aaabd99f856b7121dd85923c504a9a9ee9c5/test/CMakeLists.txt#L441

Please double check that test/CMakeLists.txt doesn't require adjustment in the current PR.

Reply:
Good catch, thanks!